### DISCUSSION TITLE
[Suggestion] Fix mekanism culling

### DISCUSSION THREAD

**Discussion by TheAirBlow** [ACGaming/UniversalTweaks#322] [Bugfix Suggestions]
**Note: Mekanism Community Edition doesn't have a fix for this.**
I had implemented 2 mixins that fix block culling in mekanism generators some time ago. I don't really know if it impacts rendering performance much, but it should be noticeable when at least ~3 big mekanism multiblocks are built.
The issue in a nutshell: the blocks used in multiblocks always have all 6 sides rendered, even if there is a block touching a side. Any minecraft block by default does obey culling and does remove a face adjacent to another non-opaque block, but mekanism devs didn't bother and they set every generator and reactor block to be opaque (aka "transparent")
Mixins: [BlockGenerator](https://github.com/TheAirBlow/MekanismCulling/blob/main/src/main/java/ml/theairblow/culling/MixinBlockGenerator.java) and [BlockReactor](https://github.com/TheAirBlow/MekanismCulling/blob/main/src/main/java/ml/theairblow/culling/MixinBlockReactor.java)

**Comment by ACGaming** [comment]
Have you considered committing to https://github.com/Thorfusion/Mekanism-Community-Edition/tree/1.12.2 yet?
